DocuSignSamplePlugin/DocuSignPlugin/pom.xml

```diff
@@ -0,0 +1,102 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.ibm.ecm.extension</groupId>
+  <artifactId>DocuSignPlugin</artifactId>
+  <packaging>jar</packaging>
+  <version>2.0</version>
+  <name>DocuSignPlugin</name>
+
+  <dependencies>
+    <!-- Navigator-->
+    <dependency>
+      <groupId>com.ibm.ecm.icn</groupId>
+      <artifactId>navigatorAPI</artifactId>
+      <version>[3.0.11-0,)</version>
+      <scope>provided</scope>
```

Comment:
This should have <systemPath> as navigatorAPI won't be available via Maven repository.